java/Nikiforov-Oleksandr/src/com/zoolatech/lecture1/tasks/_2/Task2.java

```diff
@@ -0,0 +1,26 @@
+package com.zoolatech.lecture1.tasks._2;
+import java.util.Scanner;
+
+/**
+ Write a program that accepts a distance in meters and time in seconds and outputs a speed
+ in meters/second and kilometers/hour. The result should be a floating point number.
+ */
+
+public class Task2 {
+    public static void main(String[] args) {
+        // task solution code here
+        Scanner sc = new Scanner(System.in);
+        float distance = sc.nextFloat();
+        float time = sc.nextFloat();
+        calculateSpeed(distance,time);
+    }
+
+
+    public static void  calculateSpeed(float distance, float time){
+        // calculate speed in m/sec  and in km/h
+        System.out.println("Distance in m/sec:" + distance / time);
+        System.out.println("Distance in km/h: "+distance * 3.6f / time);
+
```

Comment:
keep attention when you use extra new lines which are actually not needed here: it doesn't affect the program execution, but it might decrease the code readability